test/jsinspect_test.js

```diff
@@ -249,6 +249,31 @@ exports.jsinspect = {
           }
         );
       }
+    },
+
+    reportToFile: function(test) {
+      fs.mkdir('tmp', function () {
```

Comment:
You're creating a directory here, which won't be removed after test run.
We may want to remove it afterwards?